src/Microsoft.AspNetCore.OData/Extensions/Endpoint/ODataEndpointPattern.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if !NETSTANDARD2_0
```

Comment:
>!NETSTANDARD2_0 [](start = 4, length = 15)

This conditional is different from the other conditional. Can you explain the reason?

Reply:
Resolved.

---
In reply to: [376008587](https://github.com/OData/WebApi/pull/2035#discussion_r376008587) [](ancestors = 376008587)